core/src/main/java/de/chkal/mvctoolbox/core/translation/PropertiesTranslationResolver.java

```diff
@@ -0,0 +1,58 @@
+package de.chkal.mvctoolbox.core.translation;
+
+import java.text.MessageFormat;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.ResourceBundle;
+import javax.mvc.MvcContext;
+
+/**
+ * Resolves translations from a resource bundle with a configurable base name. The {@link Locale} for the
+ * processed request will be resolved by the applied {@link MvcContext}.
+ *
+ * @author Tobias Erdle
+ */
+public class PropertiesTranslationResolver implements TranslationResolver {
```

Comment:
We could think about moving some base functionality to a base class `AbstractTranslationResolver` so other implementations can benefit from it. Especially the MessageFormat handling. But let's do this later after this one got merged.

Reply:
:+1: